json-path-assert/build.gradle

```diff
@@ -1,7 +1,5 @@
 apply from: "$rootDir/gradle/publishMaven.gradle"
 
-displayName = "JsonPath Assert"
```

Comment:
Apparently, since Gradle 3.3, you can no longer set the display name from the build file. You can set it via `gradle.properties` if you want to, but those are in the `.gitignore` for this project.